components/ui/Button.tsx

```diff
@@ -172,6 +172,6 @@ const styles = StyleSheet.create({
     alignItems: "center",
   },
   disabled: {
-    opacity: 0.3,
+    opacity: 0.9,
```

Comment:
why was this opacity changed?

Reply:
so that even thought when we are loading we can still see the loading message since while we are loading the button is acting as disabled